python/beeai_framework/backend/constants.py

```diff
@@ -39,4 +39,9 @@ class ProviderModelDef(BaseModel):
     "watsonx": ProviderDef(name="Watsonx", module="watsonx", aliases=["watsonx", "ibm"]),
     "Groq": ProviderDef(name="Groq", module="groq", aliases=["groq"]),
     "xAI": ProviderDef(name="XAI", module="xai", aliases=["xai", "grok"]),
+    "AmazonBedrock": ProviderDef(
+        name="AmazonBedrock",
+        module="amazonbedrock",
```

Comment:
I would do `amazon_bedrock`(TS uses `amazon-bedrock`)

Reply:
```
Bedrock: {
    name: "Bedrock",
    module: "amazon-bedrock",
    aliases: ["amazon", "bedrock"] as string[],
  }
```

Reply:
I'll check again, but from memory I hit an issue in the emitter.

Reply:
Done. Changed to `amazon_bedrock` for consistency